tests/integration/test_rex.py

```diff
@@ -556,32 +553,32 @@ def test_date_functions(c):
             EXTRACT(QUARTER FROM d) AS "quarter",
             EXTRACT(SECOND FROM d) AS "second",
             EXTRACT(WEEK FROM d) AS "week",
-            EXTRACT(YEAR FROM d) AS "year"
-
-            LAST_DAY(d) as "last_day",
-
-            TIMESTAMPADD(YEAR, 2, d) as "plus_1_year",
-            TIMESTAMPADD(MONTH, 1, d) as "plus_1_month",
-            TIMESTAMPADD(WEEK, 1, d) as "plus_1_week",
-            TIMESTAMPADD(DAY, 1, d) as "plus_1_day",
-            TIMESTAMPADD(HOUR, 1, d) as "plus_1_hour",
-            TIMESTAMPADD(MINUTE, 1, d) as "plus_1_min",
-            TIMESTAMPADD(SECOND, 1, d) as "plus_1_sec",
-            TIMESTAMPADD(MICROSECOND, 999*1000, d) as "plus_999_millisec",
-            TIMESTAMPADD(MICROSECOND, 999, d) as "plus_999_microsec",
-            TIMESTAMPADD(QUARTER, 1, d) as "plus_1_qt",
-
-            CEIL(d TO DAY) as ceil_to_day,
-            CEIL(d TO HOUR) as ceil_to_hour,
-            CEIL(d TO MINUTE) as ceil_to_minute,
-            CEIL(d TO SECOND) as ceil_to_seconds,
-            CEIL(d TO MILLISECOND) as ceil_to_millisec,
-
-            FLOOR(d TO DAY) as floor_to_day,
-            FLOOR(d TO HOUR) as floor_to_hour,
-            FLOOR(d TO MINUTE) as floor_to_minute,
-            FLOOR(d TO SECOND) as floor_to_seconds,
-            FLOOR(d TO MILLISECOND) as floor_to_millisec
+            EXTRACT(YEAR FROM d) AS "year",
+
+            LAST_DAY(d) as "last_day"
+
+            -- TIMESTAMPADD(YEAR, 2, d) as "plus_1_year",
```

Comment:
https://www.w3resource.com/mysql/date-and-time-functions/mysql-timestampadd-function.php